addons/l10n_ar/demo/product_product_demo.xml

```diff
@@ -97,7 +97,8 @@
 
     <!-- VAT 10,5% -->
 
-    <record id="product.product_product_25" model="product.product">
+    <!-- TODO MATP Should remove (file name: product_product_demo, it's a product.template) -->
```

Comment:
@Demesmaeker Should I remove it ?

Reply:
The comment? Or the product? Not even sure what you are talking about here '^^
Anyway, you shouldn't have to delete a product in this PR, especially not in a localization that correct existing products, just ensure that everything is working fine and as expected.